jenkins-slaves/ansible/kie-rhel7.yml

```diff
@@ -4,7 +4,8 @@
   gather_facts: false
 
   vars :
-    maven_versions: [3.3.9,3.5.0,3.5.2,3.5.4,3.6.0,3.6.2]
+    maven_versions: [3.3.9,3.5.0,3.5.2,3.5.4,3.6.0,3.6.2,3.6.3]
+    maven_versions_needing_extension: [3.3.9,3.5.0,3.5.2,3.5.4,3.6.0]
```

Comment:
there is mentioned in the https://issues.apache.org/jira/browse/MNG-5315 that it should work without that parallel solver Maven from version 3.5.0 we could reduce the existing list to that 3.3.9 only ;)